app/frontend/styles/_application-card.scss

```diff
@@ -84,4 +84,16 @@
       margin-top: auto;
     }
   }
+
+  &__course_type {
+    margin-bottom: 10px;
+
+    font-weight: 700;
+  }
+
+  &__undergraduate {
+    padding: 10px;
```

Comment:
Rather than using 10px, we should use govuk-spacing($value). Same on line 88.

Reply:
Done